packages/docutils/lib/builder/deploy.ts

```diff
@@ -68,7 +64,14 @@ async function findDeployVersion(packageJsonPath?: string, cwd = process.cwd()):
       'No "version" field found in package.json; please add one or specify a version to deploy'
     );
   }
-  return version;
+
+  // return MAJOR.MINOR as the version by default, if that is a thing we can extract, otherwise
+  // just return the version as is
+  const versionParts = version.split('.');
+  if (versionParts.length < 2) {
```

Comment:
It's not possible for the length of `versionParts` to be `0`.

```suggestion
  if (versionParts.length === 1) {
```